pkg/streamingpromql/testdata/upstream/functions.test

```diff
@@ -446,10 +446,10 @@ clear
 
 
 # Tests for clamp_max, clamp_min(), and clamp().
-load 5m
-	test_clamp{src="clamp-a"}	-50
-	test_clamp{src="clamp-b"}	0
-	test_clamp{src="clamp-c"}	100
+# load 5m
+# 	test_clamp{src="clamp-a"}	-50
+# 	test_clamp{src="clamp-b"}	0
+# 	test_clamp{src="clamp-c"}	100
```

Comment:
This change is unnecessary - we can continue loading the data and just skip the tests below.